halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/PersistentStorageController.java

```diff
@@ -82,6 +83,49 @@
     builder.setRevert(() -> halconfigParser.undoChanges());
     builder.setSave(() -> halconfigParser.saveConfig());
 
-    return DaemonTaskHandler.submitTask(builder::build, "Edit storage settings");
+    return DaemonTaskHandler.submitTask(builder::build, "Set current persistent store");
+  }
+
+  @RequestMapping(value = "/persistentStore/{persistentStoreName:.+}", method = RequestMethod.GET)
+  DaemonTask<Halconfig, PersistentStore> getPersistentStore(@PathVariable String deploymentName,
```

Comment:
It can only be set through through the '/currentPersistentStore' (which I will refactor based on your above comment). I didn't want 'enable/disable' per store because you can only set one at a time. However, when you set 'currentPersistentStore, it gets mapped to 'enabled: true/false' for each persistent store here: https://github.com/spinnaker/halyard/pull/359/commits/3b707e0f1ac99e5ada39753d09e2aa567596b1d6#diff-b362be0f61df2de7c9543613c182b8edR111

Which brings up a larger question I have: how _exactly_ is the halconfig used? I'm only familiar with the current model in front50.yml where each persistent store has 'enabled: true/false'. If front50 switches to use the halconfig as the one source of truth (I assume that's the plan?) can front50 config be refactored to use a 'type: gcs' setting under the main persistentStorage node rather than 'enable/disable' under each child node?